app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -64,6 +75,19 @@ export const ListItemDefaults = {
     }
   },
 
+  @observes(
```

Comment:
I would REALLY prefer if we didn't have to use observers.

If you have to,  I think you can do:
@observes("bulkSelectEnabled", "topic.{pinned,visible,archived,closed,category,tags}")
```

Reply:
Okay updated. How about a compromise? I switched to using computed properties where possible. 

These were using observers already with the buffer render,
```
rerenderTriggers: ["bulkSelectEnabled", "topic.pinned"],
```

so the only one I'm really adding is for the tags.